doc-content/enterprise-only/openshift/rhpam711-kieserver-mysql-ref.adoc

```diff
@@ -112,9 +112,6 @@ https://cloud.google.com/container-engine/docs/services/[container-engine docume
 |8080 | http
 .2+| All the KIE Server web server's ports.
 |8443 | https
-.1+| `${APPLICATION_NAME}-kieserver-ping`
```

Comment:
[technical] Some docs only remove `ping` value while others remove `${APPLICATION_NAME}-kieserver-ping` and "The JGroups ping port for clustering."

Reply:
Again not a doc question but a template question